percy/README.md

```diff
@@ -0,0 +1,45 @@
+# Percy Tests
+
+Percy tests are end-to-end tests of the OTP-RR desktop, mobile, and call-taker UIs
+with screenshots taken at key steps of the user interaction.
+
+To run locally, uncomment the line in `percy.test.js` that says `headless: false`,
+then enter (Linux syntax):
+```sh
+PERCY_OTP_CONFIG_OVERRIDE=<path-to-yml> npx percy exec -- npx jest percy/percy.test.js --force-exit
```

Comment:
There is a default Percy yml that we might want to update instead? That way we won't need to supply an override

Reply:
Do you mean a YML that uses all localhosts? The geocoder would have to be mocked eventually too.

Reply:
I made the configs explicit now in `percy.yml` for the regular vs. calltaker builds.